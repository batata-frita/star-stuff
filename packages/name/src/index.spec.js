const name = require('./index')

it('should generate a simple name from a noise', () => {
  expect(name(0.7)).toMatchSnapshot()
})
