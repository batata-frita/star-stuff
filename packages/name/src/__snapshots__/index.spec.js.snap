// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should generate a simple name from a noise 1`] = `"helsaad"`;
